e2e/tests/router-navigation.spec.ts

```
@@ -0,0 +1,138 @@
+
+  test("send button and input are visible and enabled", async ({ page }) => {
+    await page.goto("/");
+    await expect(page.locator('[data-testid="user-input"]')).toBeVisible();
+    await expect(page.locator('[data-testid="send-btn"]')).toBeEnabled();
+  });
+});
+
+// Helper: open history panel and wait for sessions to load.
+async function openHistoryWithSessions(
+  page: import("@playwright/test").Page,
+) {
```

ESLint

[error] Replace `⏎··page:·import("@playwright/test").Page,⏎` with `page:·import("@playwright/test").Page`

(prettier/prettier)